tests/e2e/volume_health_test.go

```diff
@@ -1120,17 +1087,6 @@ var _ = ginkgo.Describe("Volume health check", func() {
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}()
 
-		ginkgo.By(fmt.Sprintf("Invoking QueryCNSVolumeWithResult with VolumeID: %s", volHandle))
```

Comment:
same here, why was this removed

Reply:
Same as my above comment

Reply:
volume health in CNS comes from SPBM not k8s